duckduckgo.safariextension/html/global.html

```diff
@@ -117,6 +117,15 @@
     safari.application.addEventListener("command", ddgRightClickCommand, false);
     safari.application.addEventListener("contextmenu", ddgContextMenu, false);
 
+    // use previously used localStorage['last_version'] to determine whether
+    // this is a new install or not
+    // https://github.com/duckduckgo/safari-zeroclickinfo/commit/4746f30668033551025847a33234ad15579f53d5#diff-89d925e2bf34fb6e3d713637cfd75bdbL39
+    if (typeof localStorage['last_version'] == "undefined") {
+      if (typeof localStorage['atb'] == "undefined") {
+        localStorage['atb'] = 'v' + Math.ceil((new Date().getTime() - 1456333200000) / 604800000);
+      }
+    }
```

Comment:
Yea, I'm trying to read through their docs now to understand this madness. Where is global.html loaded? Can you see it from the Web Inspector?